src/DonationForms/ServiceProvider.php

```diff
@@ -173,4 +175,14 @@ protected function registerShortcodes()
     {
         Hooks::addFilter('givewp_form_shortcode_output', GiveFormShortcode::class, '__invoke', 10, 2);
     }
+
+    /**
+     * @unreleased
+     */
+    protected function registerPostStatus()
+    {
+        add_action('init', static function () {
+            register_post_status(DonationFormStatus::UPGRADED);
```

Comment:
Does this need to use `->getValue()`? Sometimes the enum isn't processed as expected.

Reply:
No, because I'm referencing the class constant directly.

Reply:
Ah - yes. That makes sense!